### ISSUE TITLE
[Hitman] Version 1.9.0 Broke the Camera System

### ISSUE THREAD

**Issue by TheTwelfthDoctor** [FransBouma/InjectableGenericCameraSystem#18]
Just as I discovered the fact that someone made a rather awesome camera system for the newest HITMAN, version 1.9.0 that launched on January 31st came out.

I tried injecting the camera into HITMAN but it seems the update has done broke it as it gets stuck on "Waiting for camera struct interception..."

Figured I'd open an issue so this is known about.

**Comment by FransBouma** [comment]
Well, am I fast or what? ;) :P https://github.com/FransBouma/InjectableGenericCameraSystem/commit/f4719964cdd2e10ec4aa481bcbc1398dd4d75526

Yeah it was expected, they promised a patch/update on Jan 31st, so I figured it would break. I found most of it back yesterday evening and fixed the missing stuff this morning. 

the patch broke the hud toggle we use as well, so I'll make a hud toggle in just a moment and then I'll wrap it up and release an updated camera :) Stay tuned!
